doc/cephadm/adoption.rst

```diff
@@ -106,6 +107,7 @@ Adoption process
 #. Redeploy MDS daemons by telling cephadm how many daemons to run for
    each file system.  You can list file systems by name with ``ceph fs
    ls``.  For each file system::
+   This needs to be run on the master nodes.
```

Comment:
also broke the formatting:

https://docs.ceph.com/ceph-prs/35335/cephadm/adoption/#adoption-process

Reply:
still broken?